vnext/overrides.json

```diff
@@ -14,6 +14,24 @@
       "baseFile": ".flowconfig",
       "baseHash": "da1871d982a9d96c7f8235bb6c6efbb790f17cd1"
     },
+    {
+      "type": "patch",
```

Comment:
Could we attach issues to track all of these?

Reply:
I'm not sure these should have issues.  These are valid forks of the files.  -- Our color implementation will always be different than Facebooks.

Could be argued conversions.h could be split up so that we dont have so much shared code being forked there though.